docs/getting-started.md

```diff
@@ -110,6 +110,44 @@ You can now continue using your UP address within the dApp:
 const myUPAddress = myContracts.ERC725Account.address;
 ```
 
+## Options
+
+When instantiating LSPFactory options can be passed to specify parameters such as `chainId` and `ipfsGateway`.
+
+```javascript title="Instantiating LSPFactory with custom options set"
+const lspFactory = new LSPFactory(provider, {
+  deployKey: '0x...',
+  chainId: 22,
+  ipfsGateway: 'https://ipfs.infura.io:5001',
+});
+```
+
+#### Deploy Key
```

Comment:
Maybe a small table could be useful as a summary of the available options?

Like here: https://github.com/cgewecke/eth-gas-reporter#options